app/assets/stylesheets/opportunities.scss

```diff
@@ -0,0 +1,45 @@
+@import "foundation_and_overrides";
+
+.vol-panel, .opportunity-info {
+  border: 5px solid #FF851B;
```

Comment:
Color literals like `#FF851B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF851B` should be written as `#ff851b`<br>Properties should be ordered background-color, border, color, margin, padding, text-align